common/config/rush/common-versions.json

```
@@ -42,6 +42,7 @@
    */
   "allowedAlternativeVersions": {
     "@azure/ms-rest-js": ["^2.0.0"],
+    "@azure/core-http": ["^1.0.0"],
```

```suggestion
    "@azure/core-http": ["^1.0.3"],
```

If it works, I would prefer to put `^1.0.3` in the exceptions list, since `^1.0.0` is still the default for most packages.

---

rush update worked locally with ^1.0.3.

---

were there any changes to pnpm lock file when you ran the rush update? Can you commit pnpm lock file if there were changes?

---

No changes to pnpm-lock.yaml when I ran rush update